frontend/packages/dev-console/integration-tests/testData/create_user.sh

```diff
@@ -0,0 +1,54 @@
+#!/usr/bin/env bash
+
+HTPASSWD_FILE="./htpass"
+USERNAME="test"
+USERPASS="test"
+HTPASSWD_SECRET="htpasswd-test-secret"
+count=0
+sleepDuration=15
+maxRetries=40  #10 minutes
+action="started"
+
+OC_USERS_LIST="$(oc get users)"
+if echo "${OC_USERS_LIST}" | grep -q "${USERNAME}"; then
+    echo -e "\n\033[0;32m \xE2\x9C\x94 User test already exists \033[0m\n"
+    exit;
+fi
+htpasswd -cb $HTPASSWD_FILE $USERNAME $USERPASS
+
+oc get secret $HTPASSWD_SECRET -n openshift-config &> /dev/null
+
+oc create secret generic ${HTPASSWD_SECRET} --from-file=htpasswd=${HTPASSWD_FILE} -n openshift-config
+
+oc apply -f - <<EOF
+apiVersion: config.openshift.io/v1
+kind: OAuth
+metadata:
+  name: cluster
+spec:
+  identityProviders:
+  - name: test
+    challenge: true
+    login: true
+    mappingMethod: claim
+    type: HTPasswd
+    htpasswd:
+      fileData:
+        name: ${HTPASSWD_SECRET}
+EOF
+
+sleep 10s
+oc create clusterrolebinding ${USERNAME}_role1 --clusterrole=developer --user=${USERNAME}
+oc create clusterrolebinding ${USERNAME}_role2 --clusterrole=self-provisioner --user=${USERNAME}
+oc create clusterrolebinding ${USERNAME}_role2 --clusterrole=basic-user --user=${USERNAME}
+
+until [ $count -gt $maxRetries ]; do
+  ((count=count+1))
+  sleep $sleepDuration
+done
```

Comment:
Wait... This just waits.. There no extra condition to break this loop other than the count? This is different then https://github.com/openshift/console/blob/b73f9bd3f0e8626b32939707b096dac9b624187b/test-prow-e2e.sh#L26

So this will always 10 minutes (maxRetries 40 x sleepDuration 15 = 600 seconds).